ChatQuickstart/ChatViewController.swift

```diff
@@ -33,97 +28,57 @@ class ChatViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        chatManager.chatManagerDelegate = self
         
         // Listen for keyboard events and animate text field as necessary
         NotificationCenter.default.addObserver(self,
                                                selector: #selector(keyboardWillShow),
-                                               name:Notification.Name.UIKeyboardWillShow,
+                                               name:UIResponder.keyboardWillShowNotification,
                                                object: nil);
         
         NotificationCenter.default.addObserver(self,
                                                selector: #selector(keyboardDidShow),
-                                               name:Notification.Name.UIKeyboardDidShow,
+                                               name:UIResponder.keyboardDidShowNotification,
                                                object: nil);
         
         NotificationCenter.default.addObserver(self,
                                                selector: #selector(keyboardWillHide),
-                                               name:Notification.Name.UIKeyboardWillHide,
+                                               name:UIResponder.keyboardWillHideNotification,
                                                object: nil);
         
         // Set up UI controls
-        self.tableView.rowHeight = UITableViewAutomaticDimension
+        self.tableView.rowHeight = UITableView.automaticDimension
         self.tableView.estimatedRowHeight = 66.0
         self.tableView.separatorStyle = .none
     }
     
     
     override func viewDidAppear(_ animated: Bool) {
-        // Create the general channel (for public use) if it hasn't been created yet
-        
-        let options = [
-            TCHChannelOptionFriendlyName: "General Channel",
-            TCHChannelOptionType: TCHChannelType.public.rawValue
-            ] as [String : Any]
-        if let client = client, let channelsList = client.channelsList() {
-            channelsList.createChannel(options: options, completion: { channelResult, channel in
-                if (channelResult.isSuccessful()) {
-                    print("Channel created.")
-                } else {
-                    print("Channel NOT created.")
-                }
-            })
-        }
-        
-        
         super.viewDidAppear(animated)
         login()
     }
     
     override func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
-        logout()
+        chatManager.logout()
     }
     
     
     // MARK: Login / Logout
```

Comment:
Here is no logout anymore.

Reply:
There is still logout mentioned.

Reply:
This was replaced with shutdown

Reply:
But it was not replaced in this exactly `mark`.